apps/stories/management/commands/create_user_accounts.py

```diff
@@ -81,7 +81,8 @@ def handle(self, *args, **options):
                 )
             except Exception as ex:
                 boundary = None
-
+                print cluster +  " is not found." #should we continue to process this record or go to next one?
+               
```

Comment:
Well, the issue was that in the following piece of code, it should have been `cluster` instead of `district`.
```
                boundary = Boundary.objects.get(
                    name=cluster,
                    hierarchy__name="district",
                    type__name="Primary School"
                )
```
I don't think that is fixed here.